packages/ui5-application-writer/test/options.test.ts

```diff
@@ -11,9 +14,20 @@ describe('UI5 templates', () => {
         removeSync(outputDir); // even for in memory
     });
 
-    it('generates options', async () => {
-        const projectDir = join(outputDir, 'testapp_options');
-        const fs = await generate(projectDir, {
+    afterEach(() => {
+        return new Promise((resolve) => {
+            // write out the files for debugging
+            if (debug) {
+                fs.commit(resolve);
+            } else {
+                resolve(true);
+            }
+        });
+    });
+
+    it('generates simple options', async () => {
```

Comment:
Do we mean 'with default' options?

Reply:
No there are no default options. There are simple types (boolean) which simply add some file entries. Then there are options that require template params.

Reply:
You could rightly ask what about writing a value thats a boolean...well that would be: 
```
appOptions: {
    someNewProp: 'true'
}
...
<%- someNewProp %>
...
```